utils/src/treev2/quickPickWizard/QuickPickWizardContext.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../index';
+
+export function getLastNode<TNode = unknown>(context: types.QuickPickWizardContext): TNode | undefined {
+    if (context.pickedNodes.length) {
+        return context.pickedNodes[context.pickedNodes.length - 1] as TNode;
+    }
+
+    return undefined;

```

Comment:
I feel like that this should just throw an error if it was `undefined`. I pretty much see every instance of this throwing some variation of the `NoResourceFoundError` whenever it's invoked anyway.

If we need it for when `getLastNode` is the root (which I don't think we should allow users to go back to anyway), then maybe have a flag to stifle error throwing?